pkg/reconciler/common/releases.go

```diff
@@ -85,9 +98,50 @@ func componentDir(instance v1alpha1.KComponent) string {
 	return ""
 }
 
+func componentURL(version string, instance v1alpha1.KComponent) string {
+	component := "serving"
+	urlList := []string{}
+	switch instance.(type) {
+	case *v1alpha1.KnativeServing:
+		urlList = ServingYamlNames
+	case *v1alpha1.KnativeEventing:
+		component = "eventing"
+		for _, value := range EventingYamlNames {
+			if strings.Contains(value, "%s") {
```

Comment:
This seems obsolete, there's no such resource above.

Reply:
Done.